apps/ledger-live-mobile/src/screens/NotificationCenter/helper.ts

```diff
@@ -0,0 +1,51 @@
+enum TypeOfTime {
+  second = "second",
+  minute = "minute",
+  hour = "hour",
+  day = "day",
+  week = "week",
+  month = "month",
+  year = "year",
+}
+
+function getTime(timestampNew: number): [number, TypeOfTime] {
```

Comment:
we already have date-fns installed in LLM
Maybe it could be a good idea to use this ? https://date-fns.org/v2.29.3/docs/formatDistanceToNow

Reply:
Indeed that would have been great but I think that for localization reasons it would add more complexity. With the current solution we just return a number and a suffix for the "scale" of time (seconds, minutes, hours, ...) which is pretty easy to translate